index.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-var toString = {}.toString;
+var arrayObjectStringTag = '[object Array]';
+var toString = Object.prototype.toString;
 
 module.exports = Array.isArray || function (arr) {
-  return toString.call(arr) === '[object Array]';
+  return toString.call(arr) === arrayObjectStringTag;
```

Comment:
fwiw this is unlikely to be more performant (engines are very good at optimizing inline strings) and may even be slower.